app/api/composer.json

```diff
@@ -12,8 +12,7 @@
     "beberlei/doctrineextensions": "^1.0",
     "caseyamcl/guzzle_retry_middleware": "^2.7.1",
     "doctrine/doctrine-orm-module": "^5",
-    "doctrine/orm": "^2.19.5",
-    "doctrine/persistence": "^2.1",
+    "doctrine/orm": "^2.20",
```

Comment:
We're almost there :) it doesn't look like this has given us what we expected unfortunately, it's bringing in 2.20.0 and the latest is 2.20.2. Easiest way just make it "^2.20.2" and one more new lock! Everything else looking good.

Reply:
Thinking on - you can wait to change this over - we'll need a new lock file in any case once https://github.com/dvsa/olcs-transfer/pull/116 is merged